easybuild/easyconfigs/s/Singularity/Singularity-2.3.1-GCC-5.4.0-2.26.eb

```diff
@@ -22,7 +22,7 @@ sanity_check_paths = {
     'dirs': ['etc', 'libexec/singularity'],
 }
 
-# next steps after instalations
+# next steps after installation
 # INSTALATION_PATH=your_instalation_path
```

Comment:
Thanks @frenchwr and I'm a little embarrassed even to ask this, but could you fix the other misspellings of 'installation' in this comment too?